docs/source/models.rst

```diff
@@ -784,6 +784,60 @@ method to load MLflow Models with the ``prophet`` model flavor in native prophet
 
 For more information, see :py:mod:`mlflow.prophet`.
 
+Pmdarima (``pmdarima``)
```

Comment:
Can we indicate that this is experimental?

```suggestion
Pmdarima (``pmdarima``) (Experimental)
```

Reply:
yep!